lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.ts

```diff
@@ -188,15 +192,23 @@ export class ProcessListCloudComponent extends DataTableSchema implements OnChan
     skipCount: number = 0;
     currentInstanceId: string;
     selectedInstances: any[];
-    isLoading = true;
+    isLoadingProcesses = true;
+    isLoadingColumnsOrder = true;
+
+    get isLoading(): boolean {
+        return this.isLoadingProcesses || this.isLoadingColumnsOrder;
```

Comment:
Can we unify this isLoading?

Reply:
I though about it but could not find any good solution.

We need to wait with displaying table until we can create `columns schema` and after all `processes` are loaded.

`Columns schema` consist now from two factors:  `columns order` and `columns` itself.

`columns` are provided using `transclusion`, therefor creating schema should be done in `ngAfterContentInit` (and the same for loading columns order, since it is needed for creating schema)

From the other hand, processes are loaded after change of property in `ngChange` (`columns schema` is loaded only once).

In order to unify this flag I should run this both actions (load processes and create schema) in one place e.g. `ngChange` (but due to transclusion, creating schema should be done in `ngAfterContentInit`).

Reply:
You can catch the event n the latter and put all on Loading in the meantime, it makes sense to have it loading and showing the table once if finished to have its loading

Reply:
unified `isLoading` flag with `isColumnSchemaCreated$` observable